_pages/en_US/troubleshooting.txt

```diff
@@ -102,6 +102,8 @@ Your friend code was blocked from using the friend code bot because your 3DS did
 <summary><u>We were unable to successfully complete your bruteforce request. :`(</u></summary>
 
 The website has determined that your `movable.sed` cannot be brute-forced. Ensure that you gave the correct ID0 to the website. If your ID0 is correct, then you will be unable to use Seedminer and you will have to use an alternate method that will require additional games or hardware.
+
+If you do not want to format, and use a New model, indicated by 4 shoulder buttons and 2 C-Sticks, you can follow [Homebrew Launcher (super-skaterhax)](homebrew-launcher-(super-skaterhax)), an alternate method for the latest firmware.
```

Comment:
2 c-sticks...this hurts me

Reply:
that makes me happy, though what is the problem?

Reply:
one is not a cstick, its a circle pad

Reply:
uhm thats cringe naming

Reply:
do you have an idea for improvement

Reply:
2 C-sticks is a completely incorrect term. Just ask if there's a C-stick, O3DS don't have em

Alternatively explain what a C-stick is too, i.e. the pad on the right side of the console